engines/python/setup/djl_python/huggingface.py

```diff
@@ -121,20 +144,22 @@ def initialize(self, properties: dict):
         if "dtype" in properties:
             kwargs["torch_dtype"] = get_torch_dtype_from_str(
                 properties.get("dtype"))
-        self.enable_rolling_batch = properties.get("rolling_batch", None)
-        if self.enable_rolling_batch and self.enable_rolling_batch.lower(
-        ) == "false":
-            self.enable_rolling_batch = None
+        self.enable_rolling_batch = properties.get("rolling_batch") is not None
 
         if self.enable_streaming:
             self._init_model_and_tokenizer(model_id_or_path, **kwargs)
             self.initialized = True
             return
         elif self.enable_rolling_batch:
-            # TODO: Add logic to call appropriate scheduler backend for rolling batch
-            self.rolling_batch = SchedulerRollingBatch(model_id_or_path,
-                                                       self.device, properties,
-                                                       **kwargs)
+            if os.getenv('OMPI_COMM_WORLD_SIZE'):
```

Comment:
check for engine, if engine != Python and tensor_parallel!=1

Reply:
Changed.